src/pages/RepoPage/BundlesTab/BundleContent/AssetsTable/AssetsTable.spec.tsx

```diff
@@ -404,31 +407,100 @@ describe('AssetsTable', () => {
       })
     })
 
+    describe('infinite scrolls', () => {
+      it('fetches more assets when user scrolls to the bottom', async () => {
+        setup({ multipleAssets: true })
+        render(<AssetsTable />, { wrapper })
+
+        const loading = await screen.findByText('Loading')
+        mockIsIntersecting(loading, true)
+
+        const asset2 = await screen.findByText('asset-2')
+        expect(asset2).toBeInTheDocument()
+      })
+    })
+
     describe('sorting table', () => {
+      describe('sorting on asset column', () => {
+        it('sorts the table by asset', async () => {
+          const { user, mockOrdering } = setup({ multipleAssets: false })
+          render(<AssetsTable />, { wrapper })
+
+          const assetColumn = await screen.findByText('Asset')
+          await user.click(assetColumn)
+
+          await waitFor(() => expect(mockOrdering).toHaveBeenCalledWith('NAME'))
+        })
+      })
+
+      describe('sorting on type column', () => {
+        it('sorts the table by type', async () => {
+          const { user, mockOrdering } = setup({ multipleAssets: false })
+          render(<AssetsTable />, { wrapper })
+
+          const typeColumn = await screen.findByText('Type')
+          await user.click(typeColumn)
+
+          await waitFor(() => expect(mockOrdering).toHaveBeenCalledWith('TYPE'))
+        })
+      })
+
+      describe('sorting on load time column', () => {
+        it('sorts the table by size', async () => {
+          const { user, mockOrdering } = setup({ multipleAssets: false })
+          render(<AssetsTable />, { wrapper })
+
+          const loadTimeColumn = await screen.findByText(
+            'Estimated load time (3G)'
+          )
+          await user.click(loadTimeColumn)
+
+          await waitFor(() => expect(mockOrdering).toHaveBeenCalledWith('SIZE'))
```

Comment:
shouldn't this be load Time? Or still size?

Reply:
So ... load time is directly related to the size, so we just use the size

Reply:
It's not immediately clear from why that would be the case :) Maybe I just missed a comment somewhere though

Reply:
I'll add a comment in for more context